consts/consts.go

```diff
@@ -0,0 +1,6 @@
+package consts
+
+const (
+	ExpBackoffDefaultMultiplier = 1.2
```

Comment:
why we need new package here? use backoff package or move it to the pipeline settings